src/tvheadend/entity/Recording.h

```diff
@@ -172,6 +177,7 @@ namespace tvheadend
       void SetPriority(uint32_t priority) { m_priority = priority; }
 
     private:
+      uint32_t         m_enabled;
```

Comment:
IMO if this is used as a boolean we should store it as that instead of letting the inner workings of HTSP creep up to this level.